src/transit/tests.py

```diff
@@ -40,6 +40,26 @@ def no_walking_routes_in_boarding_soon(r):
     return True
 
 
+# We want to ensure that given a query string, /search does not result in an error,
+# namely "Cannot read property 'filter' of null," and instead returns a list of
+# valid suggestions - autocomplete results that are either of type 'busStop' or 'googlePlace'.
+def search_returns_suggestions(r):
+    response = r.json()
+    # Make sure response was successful
+    if not response["success"]:
```

Comment:
single quotes?

Reply:
i just copied @young-k 's double quotes, should these all be changed then?

Reply:
will defer to @young-k but iirc we had single quotes as our convention